android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,449 @@
+    data.putString("message", message);
+
+    sendEvent("closeCaption.message.received", data);
+  }
+
+  /*
+   * Chat event listeners
+   */
+  @Override
+  public void onChatMessageReceived(InMeetingChatMessage message) {
+    WritableMap data = Arguments.createMap();
+    data.putString("message", message);
+
+    sendEvent("chat.message.received", message);
+  }
+
+  @Override
+  public void onSinkAttendeeChatPriviledgeChanged(int privilege) {
+    WritableMap data = Arguments.createMap();
+    data.putString("privilege", privilege);
+
+    sendEvent("chat.attendeePrivilege.changed", data);
+  }
+
+  @Override
+  public void onSinkAllowAttendeeChatNotification(int privilege) {
+    WritableMap data = Arguments.createMap();
+    data.putString("privilege", privilege);
+
+    sendEvent("chat.allowAttendee.changed", data);
+  }
+
+
+  // InMeetingServiceListener required listeners but unused for now
+  @Override
+  public void onMeetingNeedPasswordOrDisplayName(boolean needPassword, boolean needDisplayName, InMeetingEventHandler handler) {}
+  @Override
+  public void onWebinarNeedRegister(String registerUrl) {}
+  @Override
+  public void onJoinWebinarNeedUserNameAndEmail(InMeetingEventHandler handler) {}
+  @Override
+  public void onMeetingNeedColseOtherMeeting(InMeetingEventHandler handler) {}
+  @Override
+  public void onMeetingFail(int errorCode, int internalErrorCode) {}
+  @Override
+  public void onInvalidReclaimHostkey() {}
+  @Override
+  public void onFreeMeetingReminder(boolean isHost, boolean canUpgrade, boolean isFirstGift) {}
+  @Override
+  public void onFreeMeetingUpgradeToProMeeting() {}
+  @Override
+  public void onFreeMeetingUpgradeToGiftFreeTrialStop() {}
+  @Override
+  public void onFreeMeetingUpgradeToGiftFreeTrialStart() {}
+  @Override
+  public void onFreeMeetingNeedToUpgrade(FreeMeetingNeedUpgradeType type, String gifUrl) {}
+
+  /*
+   * Helpers
+   */
+  private void sendEvent(String event, long userId) {
+    WritableMap data = Arguments.createMap();
+    data.putDouble("userId", userId);
+
+    sendEvent(event, data);
+  }
+
+  private void sendEvent(String event, long userId, AudioStatus audioStatus) {
+    WritableMap data = Arguments.createMap();
+    data.putDouble("userId", userId);
+    data.putInt("audioStatus", AudioStatus.valueOf(audioStatus));
+
+    sendEvent(event, data);
+  }
+
+  private void sendEvent(String event, long userId, VideoStatus videoStatus) {
+    WritableMap data = Arguments.createMap();
+    data.putDouble("userId", userId);
+    data.putInt("videoStatus", VideoStatus.valueOf(videoStatus));
+
+    sendEvent(event, data);
+  }
+
+  private void sendEvent(String event, long userId, SharingStatus sharingStatus) {
+    WritableMap data = Arguments.createMap();
+    // https://marketplacefront.zoom.us/sdk/meeting/android/us/zoom/sdk/SharingStatus.html
+    data.putInt("sharingStatus", SharingStatus.valueOf(sharingStatus));
+
+    sendEvent(event, data);
+  }
```

Comment:
:thinking: not sure if we need this. If we can have helper functions to access this info from code I think that `userId` should be enough... That would simplify the events